zstacklib/zstacklib/utils/multipath.py

```
@@ -65,34 +74,55 @@ def write_multipath_conf(path, blacklist=None):
                         name, value = attribute.items()[0]
                         if value.strip().strip('"') == '*':
                             attribute[name] = '.*'
-                            modified = True
+                            self.modified = True
 
                         if name == 'features' and feature_to_remove in value:
                             subsection['device'].remove(attribute)
-                            modified = True
+                            self.modified = True
 
                         if cmp(sorted(default_device['device']), sorted(subsection['device'])) == 0:
```

Ruff (0.12.2)

Undefined name `cmp`

(F821)